portfolio/src/main/webapp/newScript.js

```diff
@@ -0,0 +1,11 @@
+/**Fethces the response from the server and adds it to the page */
+async function showResponse() {
+    const responseFromServer = await fetch('/hello');
+    const myObject = await responseFromServer.json();
+
+    const message = myObject[Math.floor(Math.random() * myObject.length)]
```

Comment:
Can you add some comments about what this is doing?

Reply:
Gotcha!